docs/start/authentication/api-accounts.mdx

```diff
@@ -79,10 +79,10 @@ App-level API accounts do not come pre-configured with an access token. Each tim
 
 There is no way to manually revoke or force-regenerate app-level API account access tokens. However, either of the following actions triggers a token refresh:
 
-* When the store owner's email address changes
+* When the store owner's email address changes 
 * When you modify the app API account's OAuth scopes
 
-After one of these changes, the store owner will be prompted to review the change and reauthorize the app the next time they click the app icon in the store control panel.
+After one of these changes, the store owner will be prompted to review the change and reauthorize the app the next time they click the app icon in the store control panel. 
```

Comment:
Same here, just wanted to confirm that this line didn't need changing (ticket mentions store-level API accounts, not app-level API accounts)

Reply:
@piotrekkaminski Since you approved, I'll assume that yes,  this line doesn't need changing